frontend/src/metabase/nav/components/AppBar/AppBarToggle.styled.tsx

```diff
@@ -12,7 +12,20 @@ interface SidebarButtonProps {
 
 export const SidebarButton = styled.button<SidebarButtonProps>`
   cursor: pointer;
-  display: block;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 2.25rem;
+
+  ${({ isSmallAppBar }) =>
+    isSmallAppBar
+      ? css`
+          border-radius: 6px;
+          padding: 0.5rem 0;
+        `
+      : css`
+          padding: 1rem 0;
+        `}
```

Comment:
what does this do for us? couldn't we just do:
```suggestion
  padding: 1rem 0;
```
I'm pretty sure smallAppBar is always the same height (51px 🤪 ), so the vertical padding here isn't what's determining that.